src/binary-writer-spec.cc

```diff
@@ -200,6 +202,43 @@ void BinaryWriterSpec::WriteConst(const Const& const_) {
       break;
     }
 
+    case Type::V128: {
+      WriteString("v128");
+      WriteSeparator();
+      WriteKey("value");
+
+      v128 v = const_.v128_bits;
+
+      uint64_t digits, remainder;
+      std::ostringstream reversed_result;
+      do {
+        remainder = v.v[3];
```

Comment:
If you can, I'd prefer if this was rolled up into a 4 element loop instead.

Reply:
Rolled three into a loop but not four.